Sources/Plasma/PubUtilLib/plAvatar/plAnimStage.cpp

```diff
@@ -300,7 +300,7 @@ bool plAnimStage::ISendNotify(uint32_t notifyMask, uint32_t notifyType, plArmatu
         int stageNum = genBrain ? genBrain->GetStageNum(this) : -1;
         msg->AddMultiStageEvent(stageNum, notifyType, armature->GetTarget(0)->GetKey());
 
-        if (!genBrain || !genBrain->RelayNotifyMsg(msg))
+        if (stageNum < 0 || !genBrain || !genBrain->RelayNotifyMsg(msg))
```

Comment:
What is the goal of this change? I'm a little concerned that this may have unintended side effects because, IIRC, stage `-1` means something.